ocfweb/docs/docs/staff/procedures/toner-handling.md

```diff
@@ -0,0 +1,25 @@
+[[!meta title="Recycling Used Toner"]]
+
+OCF goes through toners quite fast, due to the heavy volume of printing from our
+users. In the past, we've let it accumulate inside the storage room which
+resulted in accidental toner spillage. The following document contains how
+to recycle the toners and hope to encourage rapid removal of the toners from OCF space.
+
+OCF staffer should keep an eye out for the accumulation of the toners, and proactively
+recycle the used toners as soon as possible.
+
+## Options to dispose the toners
+
+* We can dispose the toners by returning them to
+[Staples](https://www.staples.com/sbd/cre/marketing/sustainability-center/recycling-
+services/) down at the Shattuck Ave. They give us $2 Staples rewards per toner, and
+there is maximum limit of 10 toners per month. With the rewards, we can purchase
+snacks and etc from Staples.
```

Comment:
Staples has snacks!?

Reply:
Yeet! ty for the grammar fixes, if there's more let me know